lms/static/js/spec/learner_dashboard/collection_list_view_spec.js

```diff
@@ -29,7 +29,14 @@ define([
                             modified: '2016-03-25T13:45:21.220732Z',
                             marketing_slug: 'p_2?param=haha&test=b', 
                             id: 146,
-                            marketing_url: 'http://www.edx.org/xseries/p_2?param=haha&test=b'
+                            marketing_url: 'http://www.edx.org/xseries/p_2?param=haha&test=b',
+                            /* jshint ignore:start */
+                            banner_image_urls: [
+                                '348x116': 'http://www.edx.org/images/org1/test1',
+                                '435x145': 'http://www.edx.org/images/org1/test2',
+                                '726x242': 'http://www.edx.org/images/org1/test3'
+                            ]
+                            /* jshint ignore:end */
```

Comment:
These are the property keys sent from the Program IDA through the Programs API.
If we want to change the key names, we have two choices:
1. Update the programs API to return string names for the image of various sizes
2. In the edx-platform code, do a translation from what programs API returned to a data structure where image url keys are pure strings
@jimabramson @rlucioni What are your opinion on this point raised by @AlasdairSwan ?